pragma.md

```diff
@@ -0,0 +1,37 @@
+# pragma syntax
+## motivation
+добавляет подсказки для среды исполнения, что код может быть распараллелен или векторизован.
+
+в первом случае ускоряет выполнение за счет использования мультипоточеости без явного создания воркеров, а во втором снижает количесво обращений к памяти.
+
+> В будущем могут быть добавлены другие инструкции pragma.
```

Comment:
Adds hints to the runtime that the code can be parallelized or vectorized.

in the first case, it speeds up execution by using multi-threading without explicitly creating workers, while in the second case, it reduces the number of memory accesses.

> Other pragma instructions may be added in the future.